schemas/generate_fetch_receipt.py

```
@@ -13,12 +13,26 @@
+# which is the same reason nothing else here shells out to a bare interpreter.
+#
+# The guard is KEPT rather than deleted. It now asks a different and still worth
+# asking question: not "is the right tool on PATH" (which this prefix settles)
+# but "does the locked version still match the one this generator's flags and
+# committed output were reviewed against" — which a lockfile bump can break.
+_CODEGEN = ("uv", "run", "--group", "codegen", "datamodel-codegen")
 
 
 def generate(output: Path = OUTPUT) -> None:
     """Generate the checked-in model with the reviewed generator contract."""
     version = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)